verilog/tools/lint/verilog_lint.cc

```diff
@@ -92,7 +92,11 @@ ABSL_FLAG(
     "If true, print the description of every rule formatted for the "
     "Markdown and exit immediately. Intended for the output to be written "
     "to a snippet of Markdown.");
-
+ABSL_FLAG(
+    bool, print_rules_file, false,
+    "Print the default lint rules in a format that can be used to create a "
```

Comment:
Below, you're reading the `LinterConfigurationFromFlags()`. So that means it is not the 'default lint rules' anymore ? In that case I suggest we rephrase that as 'currrent active lint rules' or something.

Reply:
I'll fix that.